### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.3][Eclipse][Wipro]:Keyboard focus order is illogical in run on web app for containers window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1379]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app for containers.
4. "Run on web app for containers" dialog, Select "Use Exisiting Radio button" tab to "Azure" container registry" and then press down arrow keyboard focus will move from "Azure container registry" to "Web App container" 
5. Then press tab and observe the keyboard focus

**Expected Result:**
 Keyboard focus should go to use existing radio button which is present under web app for container.

**Actual Result:**
Keyboard focus is going to Docker file
**observation:** In the Same window tab up to cancel button and press 2 tabs for 2nd tab keyboard focus is going to "web app container" instead of "Azure container" registry.

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![after selecting web app for container if user presses tab keyboard focus is going to Docker file edit file in run on webapp for containers](https://user-images.githubusercontent.com/22956130/35681154-8c8420c4-0782-11e8-8b1d-0b83a2b58db8.jpg)

**Comment by Eskibear** [comment]
That's by design of Eclipse UI **ExpandBar**

`Azure Container Registry` and `Web App for Container` is at the same level, and you can switch between thme using the `up arrow` and `down arrow` keys.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533977

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. It's a legacy feature hardly ever used.
* **Quality of experience**: not impacted. A simple workaround is to use the Azure portal to deploy.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

#### Workaround
![image](https://user-images.githubusercontent.com/2351748/47148956-c0a26c00-d304-11e8-9064-bf67314627d2.png)

**Comment by umasarath52** [comment]
Discussed with Ryan and approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749810
